pvnet/models/base_model.py

```diff
@@ -350,6 +353,8 @@ def _calculate_qauntile_loss(self, y_quantiles, y):
             errors = y - y_quantiles[..., i]
             losses.append(torch.max((q - 1) * errors, q * errors).unsqueeze(-1))
         losses = 2 * torch.cat(losses, dim=2)
+        if self.use_weighted_loss:
+            losses = losses * self.weighted_losses.weights
```

Comment:
what are these weights?

Reply:
These are from the WeightedLoss class, its from here: https://github.com/openclimatefix/PVNet/blob/f1534d8c01da5eb0bf8c0b001bc36aa05ced5c00/pvnet/models/utils.py#L129